src/Security/Member.php

```diff
@@ -499,7 +499,7 @@ public function logIn($remember = false)
         $this->write();
 
         // Audit logging hook
-        $this->extend('memberLoggedIn');
+        $this->extend('afterMemberLoggedIn');
```

Comment:
renamed to `afterMemberLogin`, to be more logically compatible with the `beforeMemberLoggedIn` call